nautobot/extras/tests/test_views.py

```diff
@@ -418,6 +418,24 @@ def test_create_object_with_constrained_permission(self):
             cf.delete()
         super().test_create_object_with_constrained_permission()
 
+    @override_settings(EXEMPT_VIEW_PERMISSIONS=[])
+    @mock.patch("nautobot.core.views.generic.BulkDeleteView.perform_pre_delete")
```

Comment:
Shouldn't this be patching `CustomFieldBulkDeleteView.perform_pre_delete`?

Also, please add a comment or docstring explaining why this is needed.